src/lib/SvgTouchableMixin.ts

```diff
@@ -121,6 +123,11 @@ const touchVals = touchKeys.map(key => SvgTouchableMixin[key]);
 const numTouchKeys = touchKeys.length;
 
 export default (target: { [x: string]: unknown; state: unknown }) => {
+  const hasTouchableProperty = getHasTouchableProperty(target.props);
```

Comment:
Maybe we should apply those changes only on `web`? Or does it not change anything on native platforms?

Reply:
Thanks for catching it. Updated!